pyaedt/q3d.py

```diff
@@ -462,7 +463,7 @@ def export_matrix_data(
             Matrix Type.
             Possible Values are "Maxwell", "Spice" and "Couple".
             The default value is ``None``.
-        export_AC_DC_res : bool, optional
+        export_ac_dc_res : bool, optional
             Whether to add the AC and DC res.
             The default value is ``False``.
```

Comment:
```suggestion
            Whether to add the AC and DC resistance.
            The default is ``False``.
```
Spell out res. Is "resistance" correct?